Lib/glyphsLib/builder/user_data.py

```diff
@@ -124,8 +144,9 @@ def to_glyphs_family_user_data_from_ufo(self, ufo):
 def to_glyphs_master_user_data(self, ufo, master):
     """Set the GSFontMaster userData from the UFO master-specific lib data."""
     target_user_data = master.userData
+    special_math_keys = {GLYPHS_MATH_VARIANTS_KEY, GLYPHS_MATH_EXTENDED_SHAPE_KEY}
     for key, value in ufo.lib.items():
-        if _user_data_has_no_special_meaning(key):
+        if _user_data_has_no_special_meaning(key) and key not in special_math_keys:
```

Comment:
maybe you should add the math keys to `_user_data_has_no_special_meaning`?

Reply:
`_user_data_has_no_special_meaning` is used in multiple places and in some of them we want to skip MATH key and in others we don’t.